conf.json

```diff
@@ -51,7 +51,7 @@
     "applicationName": "Node.js MongoDB Driver API",
     "googleAnalytics": "UA-29229787-1",
     "openGraph": {
-      "title": "",
+      "title": "MongoDB API",
```

Comment:
how about the same as the `applicationName` for clarity: "Node.js MongoDB Driver API"